src/components/Home/index.js

```diff
@@ -282,6 +302,58 @@ class Home extends Component {
         );
     }
 
+    renderNotLoggedInWarning = () => {
+        const { userProfile, match } = this.props;
+        const { params } = match;
+        if (params.action !== 'edit' && params.action !== 'solve') {
+            return null;
+        }
+        if (userProfile.checking || userProfile.email) {
+            return null;
+        }
+        return (
+            <>
+                <div className={`row ${home.warningContainer}`}>
+                    <div className={home.loginWarning}>
+                        <h2 className={home.warningText}>
+                            {Locales.strings.warning}
+                            {': '}
+                        </h2>
+                        {params.action === 'solve' ? Locales.strings.return_to_your_work_later : Locales.strings.return_to_your_problem_later}
+                    </div>
+                </div>
+                <div className="row">
+                    <div className={home.shareLink}>
+                        <label htmlFor="resumeWorkUrl" className="sROnly">
+                            {Locales.strings.work_link}
+                        </label>
+                        <input
+                            id="resumeWorkUrl"
+                            type="text"
+                            aria-labelledby="resumeWorkDesc"
```

Comment:
`aria-labelledby` needs to be removed now that we have a `<label>`.

Reply:
sorry had missed it, removed it now